dask/dataframe/core.py

```diff
@@ -336,6 +335,25 @@ def quantiles(self, q):
         """
         return quantiles(self, q)
 
+    def resample(self, rule, how=None, axis=0, fill_method=None, closed=None,
+                 label=None, convention='start', kind=None, loffset=None,
+                 limit=None, base=0):
+        first, last = self.divisions[0], self.divisions[-1]
+        new_divisions = list(pd.date_range(start=first, end=last, freq=rule))
```

Comment:
If I have divisions like 

    January, February, October, November

Then it may be that my data is unevely distributed and I want to maintain this rough uneven density and number of partitions.  Would it be possible to sync up each new division to correspond with the rough location of the old one but aligned on the new grid?

This would be particularly important if our resampling frequency was substantially different from the scale of our divisions.  E.g. we might divide partitions by month but want minute-by-minute resampling.   Divisions are likely chosen ahead of time to be a good size to keep in memory; operations should endeavor to keep this number roughly the same if possible.